### PR TITLE
Mergeame cuando ya este

### PR DESCRIPTION
**Pull request by jottaxwds**

### CODE REVIEW

**Comment by boxgames1** [comment]
LGTM!

File: index.html [hunk 1]
```diff
@@ -28,12 +28,21 @@
...
+    <h1>SI! <span>🎉</span></h1>
+    <h3>Y no podría haber sido mejor!</h3>
+    <p><em>❤️ Mamá, Papá, Valeria & Víctor agradecemos las muestras de cariño recibidas ❤️</em></p>
+  </body><script type="module">
+    import confetti from "https://cdn.skypack.dev/canvas-confetti";
+    confetti();
+    window.addEventListener("click", () => {
+      confetti();
+    });
+  </script> </html>
```
Comment (boxgames1) [inline index.html:48]: shouldn't you move this to a separate file? I think this is an anti-pattern

File: index.html [hunk 2]
```diff
@@ -15,7 +15,7 @@
     --> <title>Ya Nació el Nenu?</title>
     <style>
       html {
-        background: #ff0050;
+        background: #0cd36e;
```
Comment (boxgames1) [inline index.html:18]: cant you use DS tokens instead?

**Review by boxgames1 — COMMENTED** [review]
Not that good

File: index.html (hunk 2 above)
Comment (jottaxwds) [reply index.html:18]: Sorry, the usage of acronyms is prohibited in this organisation.

File: index.html (hunk 1 above)
Comment (jottaxwds) [reply index.html:48]: The entire project is

**Comment by jottaxwds** [comment]
> LGTM!

🌈 :trollface:

File: index.html (hunk 1 above)
Comment (jottaxwds) [reply index.html:48]:
```suggestion
  </body><script type="module">
    import confetti from "https://cdn.skypack.dev/canvas-confetti";
    confetti();
    window.addEventListener("click", () => {
      confetti();
    });
  </script></html>
```

**Pull request merged**